nasa/earthdata-search
# EDSC-3045: Fixing size of leaflet controls

# Overview

### What is the feature?

Fixes an issue where the leaflet tools were blocked by the timeline

### What is the Solution?

Watch for changes in height on the element and when a change occurs, trigger a browser resize event to force the tools to recalculate their size

### What areas of the application does this impact?

Timeline

# Testing

### Reproduction steps

- **Environment for testing: Any**
- **Collection to test with: Any**

1. Add a collection to a project
2. Visit the project page
3. Click "Back to Search"
4. Visit the collection that was added to the project
5. Confirm the leaflet tools container has resized so that it is not blocked by the timeline

# Checklist
